.github/workflows/integration-tests.yml

```diff
@@ -894,3 +894,53 @@ jobs:
           cargo clean && cargo build
       - name: lv10-stats-use
         run: ${SCCACHE_PATH} --show-stats
+
+  xcode:
+    runs-on: macos-latest
+    env:
+      SCCACHE_PATH: target/debug/sccache
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v4
+
+      - name: Install rust
+        uses: ./.github/actions/rust-toolchain
+        with:
+          toolchain: "stable"
+
+      - name: Build sccache
+        run: |
+          cargo build
+
+      - name: Create wrapper script
```

Comment:
could you please add a comment explaining why a wrapper script is needed

Reply:
Its actually explained in the docs (https://github.com/mozilla/sccache/pull/2257/commits/dfe8c5be44d421452c36a90d5becc1a32e23d395#diff-8f7ae669918b1de5a217f0f28f8becd7d0f92625c4b8334b733315b8199dd102R47), but I guess I could add a comment here.

The issue is that I can't just set `CC="sccache clang++"`, because it interprets the whole `CC` variable as a command, so its searching for a file called `sccache clang++` (so space is part of the command name). If you have other ideas for a workaround, I'm all ears.